docs(manifest): Improvements to app store members (#36605)
mdn/content: files/en-us/web/manifest/categories/index.md

{{QuickLinksWithSubpages("/en-US/docs/Web/Manifest")}}

> The `categories` manifest member lets you specify one or more classifications for your web application.
> These classifications help users discover your app in distribution platforms, such as Google Play Store, where apps are organized by category.

Reviewer: Maybe a little simpler - I think we can assume readers will work out why this is good for users.

```suggestion
The `categories` manifest member lets you specify one or more classifications for your web application.
These may be used to categorize your app in distribution platforms, such as Google Play Store.
```

Author: Yeah, can be simplified but I'd like to keep the "discovery" part.
Phrased it similar to screenshots (these images help...) and description (this text helps...):
"These categories help users discover your app in app stores."

> [!NOTE]
> The `categories` member is optional and purely advisory.
> Distribution platforms may choose to ignore it and may use different values when presenting your app.

## Syntax

```json-nolint
/* Single category */
"categories": ["productivity"]

/* Multiple categories */
"categories": ["productivity", "utilities", "social"]
```

### Values

- `categories`
  - : An array of comma-separated strings, where each string represents a category name.
    The strings should be in lowercase.
    Common categories include `business`, `education`, `entertainment`, `finance`, `games`, and `photo`.

>     See the [W3C categories list](https://github.com/w3c/manifest/wiki/Categories) for more options.

Reviewer: 1. Can a string be multi word? I.e. "project management"?
2. Can any word be used? The previous version seems to indicate that perhaps you can use any word and the set in the list is just a collection of words that you might use.

Author: The list in the [specification](https://w3c.github.io/manifest-app-info/#categories-member) does have multi word category names but the [wiki link](https://github.com/w3c/manifest/wiki/Categories#standardized-categories) does not. The wiki link also lists them as "standardized", so I've left out the detail if _any value_ can be used.

Updated to:
"W3C maintains a [list of standardized categories](https://github.com/w3c/manifest/wiki/Categories), which includes common values such as `business`, `education`, `entertainment`, `finance`, `games`, and `productivity`."

Reviewer: The fact is if they use something not common then likely the app store will omit it.

I'm not going to block on this, but I'll bet that the right thing to do would be to use the strings recommended by the app stores.
